oam/install_scripts/mcs-controllernode.service.in

```diff
@@ -8,7 +8,8 @@ Type=forking
 Environment="SKIP_OAM_INIT=1"
 ExecStart=@ENGINE_BINDIR@/controllernode
 Restart=on-failure
-ExecStop=/usr/bin/env bash -c "/bin/kill -15 $MAINPID && sleep 2 && /bin/kill -9 $(pidof controllernode) || /bin/true"
+ExecStop=@ENGINE_BINDIR@/mcs-stop-controllernode.sh
+TimeoutStopSec=$(mcsGetConfig SystemConfig DBRMTimeout)
```

Comment:
Get timeout from config file

Reply:
IIRC, that's like 5 mins, is that what we want?

Reply:
nvm, I see it's been changed to 15s for some reason.